README.md

```diff
@@ -1,18 +1,648 @@
 # veritable-cloudagent
+The project aims to enable supply chains to share insights and data across multiple data platforms, enhancing the efficiency of the supply chain network. Specifically it facilitates the onboarding of new actors into a network, facilitating connections between those and allows for interaction between the actors. Interactions include issuance of credentials, acceptance of issued credentials, requesting proofs of credentials from other actors and those being able to verify those requests.
 
-This is a forked version of -> https://github.com/openwallet-foundation/credo-ts-ext/tree/main mainly focusing around OpenApi/Rest client.
 
-### Attribution
+## Table of Contents
+- [Attribution](#attribution)
+- [Setup](#setup)
+- [Getting Started](#getting-started)
+- [Development Mode](#development-mode)
+- [Environment Variables](#environment-variables)
+- [Testing](#testing)
+- [WebSocket and webhooks](#webSocket-and-webhooks)
+- [Verified DRPC](#verified-drpc)
+- [Schema Definition](#schema-definition)
+- [Demoing credential issuance and verification](#demoing-credential-issuance-and-verification)
 
-Thanks for all the hard work to everybody who contributed to the [project](https://github.com/openwallet-foundation/credo-ts-ext/tree/main)
+## Attribution
+Thanks for all the hard work to everybody who contributed to the [project](https://github.com/openwallet-foundation/credo-ts-ext/tree/main). That project is under `Apache-2.0 license`
 
-For more information read [README_VERITABLE.md](./README_VERITABLE.md)
 
-## Configuration
+## Setup
+The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI (http://localhost:3000/swagger) that is provided with the server. The OpenAPI spec can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
 
-`veritable-cloudagent` takes the following arguments to configure the `AFJ Rest agent`:
+In case `npm i` fails with the references to the `node-gyp`. Please install xcode utilities by ruuning -> `brew install xcode`. Before executing `npm i` delete your node_modules in case it has some old references `rm -rf /node_modules`
```

Comment:
`rUUning`

Reply:
thanks for a flag up, this has been updated and will reflect in the next push